src/lib/room.js

```diff
@@ -257,6 +252,7 @@ export const loadMessages = async () => {
 		if (oldMessage && oldMessage.actionsVisible !== undefined) {
 			message.actionsVisible = oldMessage.actionsVisible;
 		}
+		processMessage(message);
```

Comment:
In the `processMessage` function, it handles livechat-close and msg-type command. Just to make sure it doesn't have any side effects, can we move the logic for disableInput handling to a separate function and use it at these places.
Just concerned about hitting those livechat-close and msg-type command multiple times

Reply:
Fixed this, along with this I have added changes for reversing the order in which we receive response of load messages, since messages are received from newest to oldest and they are processed in that order, so it was necessary to process them as oldest to newest